x/rollup/tests/integration/rollup_test.go

```diff
@@ -40,14 +41,16 @@ func TestRollup(t *testing.T) {
 	l1AttributesTxBz := monomertestutils.TxToBytes(t, l1AttributesTx)
 	depositTxBz := monomertestutils.TxToBytes(t, depositTx)
 
-	depositAmount := depositTx.Value()
-	var userAddr sdk.AccAddress = depositTx.To().Bytes()
+	depositAmount := depositTx.Mint()
+	to, err := gethtypes.NewLondonSigner(depositTx.ChainId()).Sender(depositTx)
+	require.NoError(t, err)
+	var userAddr sdk.AccAddress = to.Bytes()
```

Comment:
nit: we could use `utils.EvmToCosmosAddress(sender.Bytes())` here and just inline it

Reply:
oh lol I think I wrote this initial code, imo we should still update it though if you don't mind

Reply:
We use `userAddr` more than once

Reply:
Ok sounds good. Could we still update it to use `utils.EvmToCosmosAddress(from.Bytes())`?